htdocs/main.css

```diff
@@ -684,11 +684,12 @@ tr.directentry {
     border-top: none;
     border-width: 1px;
     position: absolute;
```

Comment:
```suggestion
    position: fixed;
    right: 0;
```

I think this would be a better solution as it works irrelevant of how long the name is.

Reply:
Yes, it still works on my screen, updated the changes. I hope the help widget is positioned same place on your screen compared to your previous screenshot @skarya22 
<img width="1429" alt="Screenshot 2024-03-07 at 12 46 34 PM" src="https://github.com/aces/Loris/assets/47093379/45c49084-da78-41fa-9cc5-1e2b53ef0a2a">

Reply:
With the changes I could see it aligned with the right side of the screen no matter the name length! Thank you for updating with the changes requested